### DISCUSSION TITLE
Extension icon

### DISCUSSION THREAD

**Discussion by ghost** [aunetx/blur-my-shell#125] [Branding]
First of all, thank you so much for this wonderful extension.
I don't know if this is the right place to talk about it, I apologize if I did wrong.

Browsing the [site](https://extensions.gnome.org/) looking for extensions, I noticed that some of them had icons that [followed a kind of guideline](https://iili.io/REBYrB.png) in their design, so as I had nothing to do I decided to draw an icon to **blur my shell** following this "guideline".
I applied an icon that refers to blur, a title with the same font as the others, and added the same color as the [current icon](https://extensions.gnome.org/extension-data/icons/icon_3193_IpcML8r.png) on the site.
If you want to apply this, feel free.
By the way, I used [Figma](https://www.figma.com/) to create the icon, I left a **.fig** file in case you want to make any changes.

### Result:
![result](https://user-images.githubusercontent.com/52772425/130547698-2fef57b5-1b30-4918-b2ab-5d9dafe4a54c.png)
I created some variations according to the icons I found

PS: Don't think I'm talking bad about the current extension icon, I just figured that if there is any guideline within the extension site, you wouldn't need to waste time creating a new icon.

### Files:
[icons_png.zip](https://github.com/aunetx/blur-my-shell/files/7036230/icons_png.zip)
[icons_svg.zip](https://github.com/aunetx/blur-my-shell/files/7036231/icons_svg.zip)
[edit_icons.zip](https://github.com/aunetx/blur-my-shell/files/7036233/edit_icons.zip)

**Comment by ghost** [comment]
I made a mistake, after analyzing I found that the extensions that followed a kind of guideline actually belonged to the same creator, I apologize for my mistake.
Closing this issue.

**Comment by aunetx** [comment]
First of all, thanks a lot for the attention to this project!

You do not need to apologise in any way for this issue, actually I created the icon two days ago using gimp and being very bored, and I'm very ashamed of the result (BmS written in white over blur, really?)

So I'm very open for any help regarding the logo, and I actually quite like what you did! The only problems are:

- I think that we should keep the uppercase spelling (Blur my Shell instead of blur my shell), but I can change this very easily thanks to your figma file ;p

- I don't know if we have the right to use the exact same design than what what you showed me! I would love to keep this design, but I'm afraid that the person who created it might have some kind of copyright over the idea, or something like it (I'm very bad at copyright issues)

So if you think there will be no problem with it, then I will use this icon (with uppercase)! Else, I don't really know how to design another icon, but I'm very open to suggestions like yours :)

Thanks a lot!

**Comment by ghost** [comment]
Hello again 🤗.
Well, to avoid any kind of copyright-related issue, I created other icons for you to choose from, I hope you like some of them and if you like some, you can use them without any problems!
(If you need to change anything, I left a [.fig](https://www.figma.com/) file again)

![outline_icons](https://user-images.githubusercontent.com/52772425/130735974-fdcb4d4e-4658-4d64-929b-400b951d4d2f.png)
![filled_icons](https://user-images.githubusercontent.com/52772425/130735987-40a1379c-8418-41dd-9c94-8e317bc8f580.png)

### Files:
[outline_icons.zip](https://github.com/aunetx/blur-my-shell/files/7044068/outline_icons.zip)
[filled_icons.zip](https://github.com/aunetx/blur-my-shell/files/7044069/filled_icons.zip)
[edit_icons.zip](https://github.com/aunetx/blur-my-shell/files/7044071/edit_icons.zip)

(Sorry about my English 😅)

**Comment by aunetx** [comment]
Thanks a lot!! To be honest, it is really hard to choose between all of these masterpieces... I really can't tell which one I prefer the most!

What do you think would be the best? I think between the 5 last ones, and probably outlined?

But really, you are some kind of design genius... I'm really excited about all of this, thank you very much!!

**Comment by ghost** [comment]
Hahaha, I'm not a genius, I just get really inspired sometimes.
I'm very happy that you liked it, and to be honest, I don't know which one I would choose, not that all of them were really magnificent, it's just that I mixed everything I like and that's why it's hard to choose.
The ones I liked the most were 1, 3, 6, and 9 of the outline variant, 1, 3, and 9 for simplicity, and 6 for remembering the GNOME overview, even so, I don't see myself as having the right to choose one, after all, you created this extension, which I appreciate a lot!

**Comment by aunetx** [comment]
I will definitively choose which one I will use very soon! For the moment I use the last one (you can see it on extensions.gnome.org :p), but this is not definitive!

What is sure it that those icons really are awesome :D

**Reply by ghost** [reply]
Browsing the site, I saw the new icon, I was very happy that you liked it. :)

**Comment by aunetx** [comment]
Well, I tried and played some little with Figma (that I never used before, it's really funny!)

I just changed the size of the ninth icon (my figma link [is here](https://www.figma.com/file/dee9TBthyw06KHA94nlkLU/Blur-my-Shell-Icon?node-id=22%3A138) if you want to see/change it)

However, I now think that the seventh icon is probably the most appropriate, as the icon is centred (unlike the ninth one, which feels like it was moved to right, especially with little sizes)

So I will use it for now on! But to be honest I might change it anytime, so I think this is a good idea to keep the figma link running, especially if you have any idea or if you think another is better to use :)

**Reply by ghost** [reply]
Got it, looked at your changes in the new icon, and really liked it.
About leaving a Figma link, to be honest, I didn't even remember this possibility haha, by the way, it's really interesting! :)

**Comment by aunetx** [comment]
I converted this issue to a discussion, so we can exchange on this more easily :)

**Reply by ghost** [reply]
Good idea! :)
